solr/core/src/java/org/apache/solr/handler/admin/CoreAdminOperation.java

```diff
@@ -92,32 +82,12 @@ public enum CoreAdminOperation implements CoreAdminOp {
   CREATE_OP(
       CREATE,
       it -> {
-        assert TestInjection.injectRandomDelayInCoreCreation();
```

Comment:
take it we don't need this in the future?   Or a pattern we don't want to continue?

Reply:
Eric found this later on his own, but for anyone else, this line was moved but not removed.  See [here](https://github.com/apache/solr/pull/3054#discussion_r1925326281)